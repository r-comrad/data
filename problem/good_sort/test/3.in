105
2067640 9100786 7799350 5201268 7822120 3405201 6965436 188117 9577268 6826530 3969563 2453460 6858188 3404735 6367187 6097626 4859543 7503263 3628608 5230015 2720140 1349378 2350377 3438848 741492 5292094 7439113 1292380 4530998 5996497 6738711 8218768 1282706 5394180 4123304 9099735 19058 6226176 3696500 8497007 2286512 2445463 7436624 9686055 5656076 5738774 1515410 9376180 9581557 2246637 4720297 4391527 7700905 8777172 140693 4773526 7525628 8697603 575688 4331550 927927 3467407 4136980 7664109 1777044 8100744 3956186 3358460 9558240 8111381 242162 9711291 5621401 8781783 8272445 2594323 4005187 3458703 8590795 1356549 2486524 2158668 9292763 4816641 3921684 196800 9121090 6437414 352013 1755332 2879499 2407470 366370 7386750 3409087 5219411 3272559 8383009 3347778 7173896 2343140 6457065 335728 6156757 6072956 