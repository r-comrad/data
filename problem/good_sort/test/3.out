7700905 5230015 2720140 8100744 2350377 9100786 4530998 140693 4720297 8590795 5621401 4391527 4331550 9581557 5201268 8781783 8111381 3921684 741492 9711291 9121090 1282706 352013 7822120 8272445 3272559 6072956 9292763 242162 1292380 5292094 8383009 4123304 4773526 2343140 2453460 7503263 7173896 7664109 2594323 3404735 5394180 3405201 1515410 335728 7525628 6965436 1755332 2445463 575688 4005187 196800 3696500 2286512 7436624 2486524 6826530 2246637 4816641 1356549 9686055 6156757 7386750 366370 5656076 6226176 9376180 4136980 3956186 5996497 8697603 3467407 8497007 6437414 927927 6097626 2067640 1777044 6457065 9577268 8777172 2407470 3347778 6367187 3458703 3628608 188117 6738711 3438848 9558240 2158668 3358460 8218768 5738774 6858188 5219411 7439113 9099735 4859543 7799350 19058 3969563 1349378 3409087 2879499 